bedrock/firefox/templates/firefox/election/index.html

```diff
@@ -16,152 +16,298 @@
+                  <!-- hard code en-US because election content will not be localized -->
+                  <p><a class="mzp-c-button" href="/en-US/" data-link-type="button" data-link-name="Unsupported Browser Election Content">Other special election content</a></p>
+                </div>
+                <div id="case-outofdate" class="hidden">
+                  <p>Your version of Firefox isn’t compatible with the Election Bundle. Once you’ve updated, please return to this page for the extensions.</p>
+                  {{ download_firefox(alt_copy='Update Firefox', dom_id='btn-download-oldfx', button_color='mzp-c-button mzp-t-download') }}
+                </div>
+                <div id="case-mobile" class="hidden">
+                  <p>The Election Bundle is made just for Firefox on desktop. Visit this page on your computer to download it or add the extensions.</p>
+                  <p>Digital citizens need good info. Mozilla Corporation and Firefox can help you check your registration status, read political news curated by Pocket and get other helpful resources.</p>
+                  <!-- hard code en-US because election content will not be localized -->
+                  <p><a class="mzp-c-button" href="/en-US/">Other special election content</a></p>
+                </div>
+                <div id="case-manual" class="hidden">
+                  <p><a href="{{ url('firefox.new') }}" data-button-name="Manual installation Download Firefox">Install Firefox</a> and return to this page to install the extentions.</p>
+                </div>
+                <div id="case-funnel" class="hidden">
+                  <div id="case-funnel-windows" class="hidden">
+                    <a
+                      class="download-link button mzp-c-button mzp-t-download"
+                      href="https://download.mozilla.org/?product=firefox-election-edition&os=win&lang=en-US"
+                      data-link-type="download"
+                      data-download-os="Desktop"
+                      data-download-version="firefox election bundle win"
+                      data-download-location="election page">
+                      Download Firefox Election Bundle
+                    </a>
+                  </div>
+                  <div id="case-funnel-mac" class="hidden">
+                    <a
+                      class="download-link button mzp-c-button mzp-t-download"
+                      href="https://download.mozilla.org/?product=firefox-election-edition&os=osx&lang=en-US"
+                      data-link-type="download"
+                      data-download-os="Desktop"
+                      data-download-version="firefox election bundle mac"
+                      data-download-location="election page">
+                      Download Firefox Election Bundle
+                    </a>
+                  </div>
+                  <p><a href="{{ url('firefox.new') }}" data-button-name="just firefox">Nah, just give me Firefox</a></p>
```

Comment:
Visual nitpick: it feels like this link is too close to the button, could use a little space between them, maybe `$spacing-md` (16px) ?

![image](https://user-images.githubusercontent.com/205591/46561422-7ead0b00-c8ac-11e8-915a-1c3a726d0798.png)

Reply:
I wrapped the button in a `<p>` to give it a bottom margin. It seemed better than adding more custom styles.